src/router/index.js

```diff
@@ -30,6 +31,11 @@ export default new Router({
       path: '/account',
       name: 'account',
       component: Account
+    },
+    {
+      path: '/viewCard:id',
```

Comment:
Should this not be `/card/:id` or similar?

Reply:
indeed it should. have updated it in accordance with the `this.$route.params.id` update